sqlparse/filters/reindent.py

```diff
@@ -54,7 +54,7 @@ def nl(self, offset=0):
 
     def _next_token(self, tlist, idx=-1):
         split_words = ('FROM', 'STRAIGHT_JOIN$', 'JOIN$', 'AND', 'OR',
-                       'GROUP', 'ORDER', 'UNION', 'VALUES',
+                       'GROUP BY', 'ORDER BY', 'UNION', 'VALUES',
```

Comment:
Does this have any effect? I would assume that "GROUP" and "ORDER" are handled first so that "GROUP BY" and "ORDER BY" will never be evaluatet.

Reply:
@andialbrecht would you be able to provide more context, i.e., how does this differ in nature from `split_words` in `aligned_indent.py`?

Reply:
sorry, my bad... I've misread the diff on first sight. Everything's fine :)